src/config.rs

```diff
@@ -260,6 +398,72 @@ pub struct ArbConfig {
     pub finality: u8,
 }
 
+#[serde_as]
+#[derive(Deserialize, Serialize, Debug, Clone)]
+#[serde(crate = "serde_crate")]
+#[serde(bound(serialize = "T: Display"))]
+#[serde(bound(deserialize = "T: FromStr, T::Err: Display"))]
+pub struct TradeableAmounts<T>
+where
+    T: FromStr + Display,
+    T::Err: Display,
+{
+    /// If specified, the minimum acceptable amount to trade (>=)
+    #[serde_as(as = "Option<DisplayFromStr>")]
+    #[serde(default)]
+    pub min_amount: Option<T>,
+    /// If specified, the maximum acceptable amount to trade (<=)
+    #[serde_as(as = "Option<DisplayFromStr>")]
+    #[serde(default)]
+    pub max_amount: Option<T>,
+}
+
+impl<T> TradeableAmounts<T>
+where
+    T: FromStr + Display,
+    T::Err: Display,
+{
+    /// Map tradeable amounts into new type
+    pub fn map<U>(self) -> TradeableAmounts<U>
```

Comment:
What is this function for? I removed it and nothing broke.

Reply:
This function was needed before to convert internal types `btc::Amount` and `xmr::Amount` into real types from `bitcoin` and `monero` crates, after 16200a24e91b4859ee1b98435d0f75b021e0abfa this function was not needed anymore but I kept it as it is a generic implementation of `map` for the tradeable amounts type, so maybe someday we'll need it again.